custom_components/opnsense/device_tracker.py

```
@@ -35,257 +35,196 @@
+        new_data[TRACKED_MACS] = mac_addresses.copy()
+        hass.config_entries.async_update_entry(config_entry, data=new_data)
 
-    cem = CoordinatorEntityManager(
-        hass,
-        hass.data[DOMAIN][config_entry.entry_id][DEVICE_TRACKER_COORDINATOR],
-        config_entry,
-        process_entities_callback,
-        async_add_entities,
-    )
-    cem.process_entities()
+    _LOGGER.debug(f"[device_tracker async_setup_entry] entities: {len(entities)}")
```

Ruff

Logging statement uses f-string

(G004)